GinClientApp/Dialogs/ServerAddDlg.cs

```diff
@@ -65,7 +70,8 @@ private bool AddNewServer(string serverAlias, string webConfiguration, string gi
                 }
                 else
                 {
-                    MessageBox.Show("Error. Unable to add server " + serverAlias);
+                    MessageBox.Show("Error. Unable to add server " + serverAlias+". The configuration is wrong or the server is not responding.");
```

Comment:
```suggestion
                    MessageBox.Show("Error. Unable to add server " + serverAlias+". The configuration is invalid or the server is not responding. Please check the configuration and try again.");
```

Small rephrasing, if you agree.